app/src/sharedTest/java/org/oppia/app/splash/SplashActivityTest.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.app.splash
+
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.HomeActivity
+
+/** Tests for [SplashActivity]. */
+@RunWith(AndroidJUnit4::class)
+class SplashActivityTest {
+
+  @get:Rule
+  var activityTestRule: ActivityTestRule<SplashActivity> = ActivityTestRule(
+    SplashActivity::class.java, true,
```

Comment:
Out of curiosity, why are you changing the default launch/touch mode values? Can you provide a brief line comment here explaining that?

Also, prefer documenting the names of these parameters to help folks reading the code outside of Android Studio, e.g.:

```kotlin
@get:Rule
var activityTestRule: ActivityTestRule<SplashActivity> =
    ActivityTestRule(SplashActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false)
```

Reply:
Given comments and provided documentation links for each parameter

Reply:
Please update the code to include in-line comments naming the parameters. The documentation above doesn't replace the inline-comments. These are still beneficial since they provide context for the literal values.

Reply:
@veena14cs this is still unresolved. Note that when I provide code suggestions, I suggest actually directly applying them since that's specifically what I'm looking for.